libcxx/include/span

```diff
@@ -343,6 +347,15 @@ public:
         return __data_[__idx];
     }
 
+#  if _LIBCPP_STD_VER >= 26
+    _LIBCPP_HIDE_FROM_ABI constexpr reference at(size_type __idx) const {
+      if (__idx >= size()) {
+        __throw_out_of_range();
+      }
+      return *(data() + __idx);
```

Comment:
```suggestion
      if (__idx >= size())
        __throw_out_of_range();
      return data()[__idx];
```
Maybe also rename `__idx` to `__index` for better readability. Same below

Reply:
Honestly, that seems really weird to me. Why would you write in in such a convoluted way? This seems like a case where the wording should simply be adjusted to something simpler, or I'm missing something and there is a test missing.

Reply:
I already renamed the arg as you suggested but in the synopsis both methods would be using `idx`
```c++
constexpr reference operator[](size_type idx) const;
constexpr reference at(size_type idx) const;
```
Would it make sense to also rename the arg of `operator[]` for consistency?

Reply:
For consistency I changed the implementation to how it is implemented in the other methods. Would that be a good compromise?